coralproject/talk
# Sync Hotfix/6.16.2 back into develop to avoid future release merge conflicts

NOTE: **PLEASE DO A REGULAR MERGE** (do not squash merge this, we want to preserve commit history since this is a hotfix/release branch)

## What does this PR do?

Reintegrate the hotfix 6.16.2 back into develop to bring develop in sync with main so that future releases are good to merge and have the new release tagging in the package.json.

## How do I test this?

- Check out code
- Run a build
- Check that everything is fine and that the hotfix features are working
    - flatten replies fix
    - comment URL/pasting URL fixes
    - docker build terser package update fix to prevent memory leaks

## How do I deploy this?

Not necessary, this is a post-release sync merge.
